central/jwt/jwt.go

```diff
@@ -19,8 +21,9 @@ var (
 )
 
 const (
-	privateKeyPath    = "/run/secrets/stackrox.io/jwt/jwt-key.der"
-	privateKeyPathPEM = "/run/secrets/stackrox.io/jwt/jwt-key.pem"
+	privateKeyDir     = "/run/secrets/stackrox.io/jwt"
+	privateKeyPath    = privateKeyDir + "/jwt-key.der"
```

Comment:
nit: maybe also rename this to privateKeyPathDER? At the very least it's consistent with the PEM naming.